### PR TITLE
feat(tooltip): initial version of the tooltip directive

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**

### CODE REVIEW

**Comment by pkozlowski-opensource** [comment]
Yay! Initial version of a tooltip directive! It is bare-bones right now, but all the essentials are in place :-) Next steps:
* [ ] - figure out positioning
* [ ] - create popovers (will be almost exactly the same so need to figure out code sharing / refactoring)

**Comment by wesleycho** [comment]
This is really nice work! LGTM

**Comment by pkozlowski-opensource** [comment]
Thnx @wesleycho ! I will just move positioning to `ngAfterViewChecked` and merge.

File: demo/src/app/shared/side-nav/side-nav.component.html [hunk 1]
```diff
@@ -49,6 +49,9 @@
           <li>
             <a [routerLink]="['/components/tabs']">Tabs</a>
           </li>
+          <li>
```
Comment (icfantv) [inline demo/src/app/shared/side-nav/side-nav.component.html]: `<li>` needs `routerLinkActive`

**Pull request closed without merging**